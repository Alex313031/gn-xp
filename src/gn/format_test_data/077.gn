# Force sorting based on comment.

# SORT
foo = [
  "z",
  "a",
]

# SORT
# This is SORT because reason.
bar -= [
  "z",
  "z2",
  "a",
  "y.cc",
]

# This is SORT because reason:
# SORT
baz += [
  "z",
  "z2",
  "a",
  "y.cc",
]
