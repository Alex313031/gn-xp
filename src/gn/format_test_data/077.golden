# Force sorting based on comment.

# SORT
foo = [
  "a",
  "z",
]

# SORT
# This is SORT because reason.
bar -= [
  "a",
  "y.cc",
  "z",
  "z2",
]

# This is SORT because reason:
# SORT
baz += [
  "a",
  "y.cc",
  "z",
  "z2",
]
